<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MasterForge</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>551</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="rightClickLabel">
        <property name="font">
         <font>
          <family>.SF NS Text</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Right Click: Scene/Room Menu, Double click room: Properties&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="font">
         <font>
          <family>.SF NS Text</family>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionRoom"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionBig_Brother"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdd"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open Workspace</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Workspace</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close Workspace</string>
   </property>
  </action>
  <action name="actionRoom">
   <property name="text">
    <string>Room</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Map Data</string>
   </property>
  </action>
  <action name="actionBig_Brother">
   <property name="text">
    <string>Big Brother</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save Workspace As</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
